documentation/docs/decentralized_identifiers/overview.md

```diff
@@ -9,68 +9,69 @@ keywords:
 - Decentralized Identifiers
 - overview
 - DLT
+- tutorial
 ---
 
 # Decentralized Identifiers (DID)
 
-The Decentralized Identifiers (DID) standard from the World Wide Web Consortium (W3C) is the fundamental standard that supports the concept of a decentralized digital identity. A DID is a unique identifier that contains information that can be resolved to a DID Document. This document contains data such as public keys, enabling the holder to prove ownership over their personal data, but also URIs that link to public information about the identity. This implementation complies to the [DID specifications v1.0 Working](https://www.w3.org/TR/did-core//).
+The Decentralized Identifiers (DID) standard from the World Wide Web Consortium (W3C) is the fundamental standard that supports the concept of a decentralized digital identity. A DID is a unique identifier that contains information that can be resolved to a DID Document. This document contains data such as public keys, enabling the holder to prove ownership over their personal data, but also URIs that link to public information about the identity. This implementation complies with the [DID specifications v1.0 Working](https://www.w3.org/TR/did-core//).
 
-In the IOTA Identity framework, we have implemented the DID standard according to the `iota` [DID Method Specification](../specs/did/iota_did_method_spec.md). We recommend seeing the `iota` DID Method Specification as the golden standard for DID on IOTA. Other implementations of DID on IOTA are recommended to follow the `iota` DID Method Specification. However, it is not necessary to implement a novel Method implementation for every project, so feel free to utilize this framework directly. 
+In the IOTA Identity framework, we have implemented the DID standard according to the `iota` [DID Method Specification](../specs/did/iota_did_method_spec.md). We recommend reading the `iota` DID Method Specification as the golden standard for DID on IOTA. Other implementations of DID on IOTA also follow the `iota` DID Method Specification when applicable. However, it is not necessary to implement a novel Method implementation for every project, so feel free to utilize this framework directly. 
```

Comment:
Tbh I have a problem to understand what it is supposed to mean. Does it mean that if I implement my own DID I should follow the `iota` DID Method Specification? Or does it mean that there are other implementations and they are most likely follow this specification, but might not?

Reply:
Yes to the first question and yes to the second; the "where applicable" is supposed to say, "Yes, the DID Method is used for a lot of implementations" but some may have something more "technical" or process driven that don't allow them to adopt it. If that makes sense.

Does the original work better for you, as a person going in this relatively "blind"?

Reply:
We position the `iota` DID method indeed as the one method to rule them all inside the IOTA ecosystem. But we can't force it. So people are allowed to use their own. Overall, we just advice to use the framework and don't bother thinking about the method and implementing it yourself as we have done that in a really good way ;) I hope that might help clarify.